faiss/python/__init__.py

```diff
@@ -189,6 +189,9 @@ def replacement_function(*args):
 add_ref_in_constructor(BufferedIOReader, 0)
 
 add_ref_in_constructor(IDSelectorNot, 0)
+add_ref_in_constructor(IDSelectorAnd, 0)
```

Comment:
Updated with slice(2), having trouble running these tests locally so currently having to rely on CI to check.

QQ but why wouldn't the IDSelectorNot follow a similar pattern or since it only needs one reference a single 0 is sufficient?

Reply:
Right about IDSelectorNot